setup.py

```diff
@@ -172,7 +175,8 @@ def get_ext_modules():
                                        "numba/cext/dictobject.c",
                                        "numba/cext/listobject.c",
                                        ],
-                              extra_link_args=install_name_tool_fixer,
+                              extra_link_args=install_name_tool_fixer +
+                              extra_link_args,
```

Comment:
NOTE: it's `_random.c` that needs `pthread` as it uses the fork handler.

Reply:
Should that be included as a comment?

Reply:
Yes, I think that's a good idea, saves searching next time. Thanks!

Reply:
done in dd33b3da3caac8d511aa5ceb20485e88e85d2c90